feedback-module/src/App.js

```diff
@@ -16,6 +16,7 @@ function App({ pageTitle, lang, endpoint }) {
   return (
     <>
       <Helmet>
```

Comment:
Don't we want the absolute URL rather than just /main.css?

Reply:
yep— just changed it to /main.css back when I was testing local changes in the styling and forgot to change it back. Going to do that now